deployments/mainnet/.chainId

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Still not super keen on having an entire file dedicated to a single integer. Is this absolutely necessary?

Reply:
It's generated from hardhat, so I think we would have to change it there.

Reply:
Hardhat-deploy has a [nice write-up](https://hardhat.org/plugins/hardhat-deploy.html#migrating-existing-deployment-to-hardhat-deploy) on why they have this file:

>The reason why hardhat-deploy save chainId in the .chainId file is both for
>
>  -  safety: so that if you were to change the network to point to a different chain, it would not attempt to read the wrong folder and assume that a contract has been deployed while it has not.
>  -  ability to know the chainId without requring to be connected to a node (and so not dependent on hardhat.config.js settings). Useful for `export` task.